multiversx_sdk_cli/transactions.py

```diff
@@ -122,3 +125,56 @@ def load_transaction_from_file(f: TextIO) -> Transaction:
     data_json: bytes = f.read().encode()
     transaction_dictionary = json.loads(data_json).get("tx") or json.loads(data_json).get("emittedTransaction")
     return Transaction.new_from_dictionary(transaction_dictionary)
+
+
+def transaction_event_to_dictionary(event: TransactionEvent) -> dict[str, Any]:
+    return {
+        "address": event.address.to_bech32(),
+        "identifier": event.identifier,
+        "topics": [topic.hex() for topic in event.topics],
+        "data": event.data.decode(),
+        "additional_data": [data.hex() for data in event.additional_data],
+    }
+
+
+def transaction_logs_to_dictionary(logs: TransactionLogs) -> dict[str, Any]:
+    return {
+        "address": logs.address.to_bech32(),
+        "events": [transaction_event_to_dictionary(event) for event in logs.events],
+    }
+
+
+def smart_contract_result_to_dictionary(result: SmartContractResult) -> dict[str, Any]:
+    return {
+        "sender": result.sender.to_bech32(),
+        "receiver": result.receiver.to_bech32(),
+        "data": result.data.decode(),
+        "logs": transaction_logs_to_dictionary(result.logs),
+    }
+
+
+def transaction_on_network_to_dictionary(tx: TransactionOnNetwork) -> dict[str, Any]:
+    return {
+        "sender": tx.sender.to_bech32(),
+        "receiver": tx.receiver.to_bech32(),
+        "hash": tx.hash.hex(),
+        "nonce": tx.nonce,
+        "round": tx.round,
+        "epoch": tx.epoch,
+        "timestamp": tx.timestamp,
+        "block_hash": tx.block_hash.hex(),
+        "miniblock_hash": tx.miniblock_hash.hex(),
+        "sender_shard": tx.sender_shard,
+        "receiver_shard": tx.receiver_shard,
+        "value": tx.value,
+        "gas_limit": tx.gas_limit,
+        "gas_price": tx.gas_price,
```

Comment:
Maybe use the "standard" JSON naming conventions / camel-case.

Reply:
done